pkg/metainfo/kvmetainfo/objects.go

```diff
@@ -40,9 +40,10 @@ var DefaultRS = storj.RedundancyScheme{
 }
 
 // DefaultES default values for EncryptionScheme
+// BlockSize should default to the size of a stripe
 var DefaultES = storj.EncryptionScheme{
 	Cipher:    storj.AESGCM,
-	BlockSize: 1 * memory.KiB.Int32(),
+	BlockSize: DefaultRS.StripeSize(),
```

Comment:
delayed comment but might be worth making this maybe double the stripe size? every encryption block gets some padding overhead. for a 1MB file, with a 1K blocksize, we end up actually storing 1MB + 16K. If we do a 2*DefaultRS.StripeSize() (where StripeSize() is 1K), then the overhead is only 8K instead of 16K.

Reply:
i'm have a different branch with some changes that relate to this.  I was thinking since the `BlockSize` config option is removed, that I would replace it with a `StripesPerBlock` config setting like so ...

```
  type EncryptionConfig struct {
[...]
+      StripesPerBlock int    `help:"the integer of how many stripes should fit in an encryption block" default:"2"
````

Reply:
oh that's a neat idea

Reply:
orrr we can just hardcode 2 stripes per block and call it good.  Im cool with either.

Reply:
shrug emoji

Reply:
i think we should default to a multiple, but i don't care strongly more than that about if it's hardcoded or configurable or what. i do think it's valuable to not have people assume it's just the stripe size always

Reply:
I prefer fewer config options when possible, so if you are /shrug about it, i will hardcode it.  I will gete a PR up with that change in a bit here and will run it by you.

Reply:
i am /shrug about it